client/objects/GitHubRepo.php

```diff
@@ -30,6 +30,7 @@ protected function getAttributes()
 			'pushed_at' => 'string',
 			'created_at' => 'string',
 			'updated_at' => 'string',
+			'parent' => 'GitHubFullRepo',
```

Comment:
To avoid recursion, please set parent attribute to be of type GitHubSimpleRepo, or implement that parent attribute only in GitHubFullRepo and set it to be of type GitHubRepo.